cli_tools/gce_vm_image_import/main.go

```diff
@@ -199,11 +199,15 @@ func buildDaisyVars(translateWorkflowPath string) map[string]string {
 	}
 	varMap["family"] = *family
 	varMap["description"] = *description
-	if *network != "" {
-		varMap["import_network"] = fmt.Sprintf("global/networks/%v", *network)
-	}
 	if *subnet != "" {
 		varMap["import_subnet"] = fmt.Sprintf("regions/%v/subnetworks/%v", *region, *subnet)
+		// When subnet is set, we need to grant a value to network to avoid fallback to default
```

Comment:
looks like network/subnet processing is the same for import and export wrappers. Consider refactoring to avoid code duplication.

Reply:
There will be a major refactoring to CLI tools as preparation of API project (so that API can call it as a lib). Will consider that by then.